plugins/headlamp-plugin/src/stateless/index.ts

```diff
@@ -0,0 +1,431 @@
+import * as jsyaml from 'js-yaml';
```

Comment:
Is this file supposed to be included?

Reply:
AH I looked through and didn't find this used anywhere, I believe it may have been created when running `npm audit fix` (I noticed it created some redundant unused files), thanks for picking this up